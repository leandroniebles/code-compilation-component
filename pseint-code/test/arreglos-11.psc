funcion error_dims(n)
  dimension a(n);
finfuncion
proceso foo
  leer n;
  error_dims(n);
finproceso