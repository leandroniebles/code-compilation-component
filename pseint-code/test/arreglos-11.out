> hola
=== Line 2: ExeError 122
